tests/api_resources/test_issuer_fraud_record.py

```diff
@@ -0,0 +1,25 @@
+import stripe
+from tests.helper import StripeTestCase
+
+
+TEST_RESOURCE_ID = 'issfr_123'
+TEST_CHARGE_ID = 'ch_123'
```

Comment:
Can we get rid of `TEST_CHARGE_ID`? I don't think it's used for anything anymore.

ptal @fay-stripe

Reply:
yup saw it right after I assigned you :)